ci/docker/ubuntu-swift.dockerfile

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM swift:5.7.3
+
+RUN apt-get update -y -q && \
+    apt-get clean 
```

Comment:
I think that this `RUN` does nothing.
Can we remove this? Furthermore, can we use `swift:5.7.3` directly?

Reply:
I removed the RUN command.  I tried using swift:5.7.3 in the docker-compose file but it fails as it requires a url to the repo I believe.  Looks like the from in the swift.dockerfile is able to resolve the url without having to hard code it.

Reply:
I've pushed a change to remove this file.

Reply:
Nice, thank you!